### ISSUE TITLE
Triangulation Question.

### ISSUE THREAD

**Issue by hwkim0325** [zippy84/vtkbool#38]
I want to triangulate every point, how can I solve this problem?

![image](https://user-images.githubusercontent.com/40055222/101738837-143a2a00-3b0a-11eb-9fc0-56ba60529d1b.png)
![image](https://user-images.githubusercontent.com/40055222/101738852-18664780-3b0a-11eb-90a7-52d2bc9c4945.png)

**Comment by MoriatyP** [comment]
Have you solved this problem？

**Comment by hwkim0325** [comment]
> Have you solved this problem？

Yes,I used another method for combining two meshes. I thought that Boolean operation was not proper algorithm to my project. So I projected points of a cylinder mesh to the other mesh in order to create points. After that I connected the points between two meshes and I triangulated them.
